src/NUnitFramework/tests/Internal/UniqueTestIdTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace NUnit.Framework.Internal
+{
+    [TestFixture]
+    public class UniqueTestIdTests
+    {
+        private static ConcurrentDictionary<string, string> _idHolder = new ConcurrentDictionary<string, string>();
```

Comment:
Minor nit here, but our coding convention is that the underscore is for private members not statics. This also does not need to be static in NUnit, the class is only created once. Please make it and the ID property non-static.

Reply:
Done!